src/styles/ovc/social-icons.css

```
@@ -0,0 +1,12 @@
+.text-dark-social {
+    color: #343a40!important;
+}
+
+.text-dark-social:hover,
+.text-dark-social:focus {
+    color: #121416!important;
+    background: none !important;
+    text-decoration-color: none !important;
```

```suggestion
    text-decoration: none !important;
```

I'm noticing an underline flash whenever you hover over an icon and then move away. It also appears when you tab from one icon to the next. This is on Windows in both Chrome and Firefox. Perhaps the above suggestion might resolve this.

---

thanks!

---

have tried that change and it still is there.